guide/multiple-wallets.md

```diff
@@ -0,0 +1,313 @@
+
+{% include image-gallery.html pages = page.images_payments %}
+
+## Balance & activity
+
+By default, balance and [activity]({{ '/guide/daily-spending-wallet/activity/' | relative_url }}) only include information from the currently selected wallet. A wallet switcher is available again right above the balance. This switcher includes the option to select multiple (or all) wallets, so users can more easily get an overview of all their funds. This selection carries through to the search functionality.
+
+{% include image-gallery.html pages = page.images_activity %}
+
+## Sending
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/multiple-wallets/send.png"
+   retina = "/assets/images/guide/multiple-wallets/send@2x.png"
+   modalImage = "/assets/images/guide/multiple-wallets/send-big.png"
+   alt-text = "Send screen with a wallet selector option"
+   caption = "Users can choose which wallet to send from"
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+A wallet indicator and switcher in the [send flow]({{ '/guide/daily-spending-wallet/sending/' | relative_url }}) is a necessity to keep users informed about which funds will be transferred. Applications may automatically switch wallets to match given payment information. For example, if the user selects a contact that only has a lightning address, the application should switch to the lightning wallet.
+
+Users should be able to easily override these if they want to act differently based on their current needs and goals. For example, an application may decide that the on-chain wallet is most appropriate for a payment because the lightning wallet is low on liquidity. But the user might be in a rush and prefers to use lightning even if it means paying a swap fee for the additional liquidity.
+
+Note that it is not possible in this design to transfer funds from multiple wallets at once.
+
+</div>
+
+## Transfers
+
+In addition to the new "from" selector, the "to" field also needs to be become more flexible to allow for transfers.
+
+Transfers between wallets on different layers include additional complexity and may require either an involved manual process, or use of a third-party service (like a [lightning service provider]({{ '/guide/how-it-works/lightning-services/' | relative_url }})).
+
+{% include image-gallery.html pages = page.images_transfer %}
+
+## Receiving
+
+Payment requests the user shares include the currently selected wallet as well. And when receiving bitcoin, ensure users can easily see which wallet the payment was received to, and navigate there with a tap.
```

Comment:
```suggestion
Payment requests the user shares include the currently selected wallet as well. When receiving bitcoin ensure that user can quickly tap to navigate there to see which wallet received the payment.
```

Reply:
I find the changed version less precise and harder to read.